### PR TITLE
adds functional and styled Filter form component

### PR DESCRIPTION
**Pull request by lgrees**
### Assignee Tasks

* [ ] added unit tests (or none needed)
* [ ] written relevant docs (or none needed)
* [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

_Your PR Notes Here_

### CODE REVIEW

File: client/components/filter-form.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+              name="category"
+              label={category.name}
+              value={category.id}
+              onChange={this.handleUpdate}
+            />
+          ))}
+        </Form.Group>
+
+        <Button>
+          <Icon name="filter" />
```
Comment (lotust) [inline client/components/filter-form.js]: please correct me if i'm wrong but won't we need Button to have a type submit?

File: client/components/filter-form.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+              control="input"
+              name="category"
+              label={category.name}
+              value={category.id}
+              onChange={this.handleUpdate}
+            />
+          ))}
+        </Form.Group>
+
+        <Button>
```
Comment (pconconroy) [inline client/components/filter-form.js]: `<Button type="submit">`

File: client/routes.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ class Routes extends Component {
           </Switch>
         )}
         {/* Displays our Login component as a fallback */}
-        <Route component={Login} />
+        <Route component={FilterForm} />
```
Comment (pconconroy) [inline client/routes.js]: `<Route component={Login} />`

File: client/components/filter-form.js (hunk 1 above)
Comment (lgrees) [reply client/components/filter-form.js]: The default is submit but I will add this for clarity

File: client/routes.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (lgrees) [reply client/routes.js]: Yep this was just for testing I will remove this before the merge

**Pull request merged**